module-1.md

```diff
@@ -0,0 +1,35 @@
+# 1: What is mesh?
+
+Total length:  1.5 hour
+Planned:      1.25 hour
+
+| **Section**                                        | **Length** |
+|----------------------------------------------------|------------|
+| **1: Topic introduction**                          | 15 min     |
+| **2: Hands-on and material exploration**           | 40 min     |
+| **3: Discussion and reflection activity**          | 20 min     |
+| **4: Wrap-up and week ahead**                      | 5 min      |
+
+## 1: Topic introduction
+
+- Discuss familiar topics like net neutrality, internet service providers, and internet censorship
+- Discuss social, political and technical reasons for a mesh
+- Succinct high-level slides on many pieces that will be explored in more detail in upcoming modules:
+    - two minutes on historic Toronto networks
+    - two minutes on community mesh networks in North America and other parts of the world
+    - general landscape of technologies like flashing routers with OpenWrt to install some mesh routing software
+- Set realistic expectations of the state of mesh technology
+
+## 2: Hands-on and material exploration
+
+- Introduce yourself, why you are interested in this workshop
+- Activity for audience to "run their own app" on pre-configured nodes, where the apps are relatable with those we are familiar with from the Internet (e.g. Facebook, Dropbox, Whatsapp, Twitter, Spotify)
```

Comment:
I think we need to figure out the best way to specify an activity, my sense: 
1. format (e.g. group, pairs)
1. materials (including slides etc...)
1. objectives

Reply:
Makes sense to me. Let's add to template.